elements/lisk-p2p/test/integration/limited_inbound_outbound_connections.ts

```diff
@@ -20,10 +20,10 @@ import { platform } from 'os';
 describe('Limited number of outbound/inbound connections', () => {
 	let p2pNodeList: ReadonlyArray<P2P> = [];
 	const NETWORK_START_PORT = 5000;
-	const NETWORK_PEER_COUNT_WITH_LIMIT = 30;
+	const NETWORK_PEER_COUNT = 10;
```

Comment:
Why reducing the number of peers? If we do this, we must check the expectations again.

Reply:
Please, just re-check the expectations of these tests. I think we need to change them now.

Reply:
I will keep number of nodes as 10 and reduce `LIMITED_CONNECTIONS` instead, to account for any concerns.